elliottlib/cli/find_bugs_kernel_cli.py

```diff
@@ -0,0 +1,335 @@
+import re
+import sys
+from typing import Any, Dict, List, Optional, Sequence, TextIO, Tuple, cast
+
+import click
+import koji
+from bugzilla import Bugzilla
+from bugzilla.bug import Bug
+from jira import JIRA, Issue
+
+from elliottlib import Runtime, brew
+from elliottlib.assembly import AssemblyTypes
+from elliottlib.cli.common import cli, click_coroutine
+from elliottlib.config_model import KernelBugSweepConfig
+from elliottlib.exceptions import ElliottFatalError
+from elliottlib.util import green_print
+
+
+class FindBugsKernelCli:
+    def __init__(self, runtime: Runtime, trackers: Sequence[str],
+                 clone: bool, reconcile: bool, comment: bool, dry_run: bool):
+        self._runtime = runtime
+        self._logger = runtime.logger
+        self.trackers = list(trackers)
+        self.clone = clone
+        self.reconcile = reconcile
+        self.comment = comment
+        self.dry_run = dry_run
+        self._id_bugs: Dict[int, Bug] = {}  # cache for kernel bug; key is bug_id, value is Bug object
+        self._bug_trackers: Dict[int, Issue] = {}  # bug_id -> KMAINT jira mapping
```

Comment:
We have an instance of bug_trackers having a specific meaning, `runtime.bug_trackers` , so would suggest a slightly different name

Reply:
what name do you suggest here?